bitnami/spark/templates/NOTES.txt

```diff
@@ -1,21 +1,29 @@
 1. Get the Spark master WebUI URL by running these commands:
 {{- if .Values.ingress.enabled }}
+
   export HOSTNAME=$(kubectl get ingress --namespace {{ .Release.Namespace }} {{ include "spark.fullname" . }}-ingress -o jsonpath='{.spec.rules[0].host}')
   echo "Spark-master URL: http://$HOSTNAME/"
-{{- else }}
+
+{{- else -}}
```

Comment:
Why did you addd the `-}}` at the end of this 'else'? That's likely to remove too many lines from the NOTES.txt output

Reply:
Right, I did intentionally in order to prettify the output.